x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/detection_rules_client/detection_rules_client_interface.ts

```diff
@@ -5,34 +5,33 @@
  * 2.0.
  */
 
-import type { RuleCreateProps } from '../../../../../../common/api/detection_engine/model/rule_schema';
-import type { PrebuiltRuleAsset } from '../../../prebuilt_rules';
 import type {
+  RuleCreateProps,
   RuleUpdateProps,
   RulePatchProps,
   RuleObjectId,
   RuleToImport,
 } from '../../../../../../common/api/detection_engine';
 import type { RuleAlertType } from '../../../rule_schema';
+import type { PrebuiltRuleAsset } from '../../../prebuilt_rules';
+import type { RuleResponse } from '../../../../../../common/api/detection_engine/model/rule_schema';
```

Comment:
Nit: Can we import `RuleResponse` along with the other imports `from '../../../../../../common/api/detection_engine';`?

Reply:
Thanks! Fixed in [`3d37bd6`](https://github.com/elastic/kibana/pull/185748/commits/3d37bd6c45b31abe06685280546a4d93997d64de)